pyproject.toml

```diff
@@ -15,6 +15,7 @@ gunicorn = "^20.1.0"
 pymongo = "^4.1.1"
 mongomock = "^4.1.2"
 pytest-env = "^0.6.2"
+Flask-Login = "^0.6.2"
 
 [tool.poetry.dev-dependencies]
 
```

Comment:
I know this already exists, but the `[tool.poetry.group.dev.dependencies]` section breaks my poetry installation - if I were to guess, this is valid but relies on a more modern version of poetry than your "requires" section claims (0.12). Is there any advantage to using that over just `[tool.poetry.dev-dependencies]`?